federation-2/router-bridge/js-src/plan.ts

```diff
@@ -148,6 +148,7 @@ export class BridgeQueryPlanner {
         },
         errors: [
           {
+            message: e.toString(),
```

Comment:
@jesse-apollo WDYT? we can keep this to get more detailed errors in case of a throw that doesn't have a message, but the tradeoff is we might leak implementation detail.

Reply:
What sort of implementation detail would be potentially leaked here?

Reply:
Errors from a call to the query planner `plan()` function (eg: missing crypto.getRandomValues() would make it all the way to the client (in the graphql errors array).

Reply:
Is there a way to get this to the Router log but not to the client?

Reply:
we could log this using the deno logger functions, and redact the messages and only keep the extensions.code, possibly as an error because this specific step (planning) should happen after validation.

Reply:
Fixed in f4edcf8

Reply:
Opened https://github.com/apollographql/federation-rs/issues/282